extensions/ql-vscode/src/view/RawTableValue.tsx

```diff
@@ -1,22 +1,30 @@
 import * as React from 'react';
 
-import { ResultValue } from '../adapt';
 import { renderLocation } from './result-table-utils';
+import { ColumnValue } from '../bqrs-cli-types';
+import { isStringLoc, isWholeFileLoc, isLineColumnLoc } from '../bqrs-utils';
 
 interface Props {
-  value: ResultValue;
+  value: ColumnValue;
   databaseUri: string;
 }
 
 export default function RawTableValue(props: Props): JSX.Element {
   const v = props.value;
-  if (typeof v === 'string') {
+  if (typeof v === 'string'
+    || typeof v === 'number'
+    || typeof v === 'boolean') {
     return <span>{v}</span>;
   }
-  else if ('uri' in v) {
-    return <a href={v.uri}>{v.uri}</a>;
-  }
-  else {
-    return renderLocation(v.location, v.label, props.databaseUri);
+
```

Comment:
(1) I fear that the fallthrough to silently rendering `<span />` when the location is not one of the known forms might lead to confusing debugging later. At least log what `loc` was, maybe?
(2) Can this logic be factored out as its own function in some way that can be shared with `renderSarifLocationWithText` and `renderSarifLocation`?

Reply:
I am trying to reproduce the logic of #546. When the location is "empty", we should render as an empty span. I can refactor to make this more explicit.